src/ray/raylet/node_manager.cc

```diff
@@ -1032,13 +1082,11 @@ void NodeManager::AssignTask(Task &task) {
   RAY_LOG(DEBUG) << "Assigning task to worker with pid " << worker->Pid();
   flatbuffers::FlatBufferBuilder fbb;
 
-  const ClientID &my_client_id = gcs_client_->client_table().GetLocalClientId();
-
   // Resource accounting: acquire resources for the assigned task.
   auto acquired_resources =
       local_available_resources_.Acquire(spec.GetRequiredResources());
-  RAY_CHECK(
-      this->cluster_resource_map_[my_client_id].Acquire(spec.GetRequiredResources()));
+//  RAY_CHECK(
+//      this->cluster_resource_map_[my_client_id].Acquire(spec.GetRequiredResources()));
```

Comment:
Why is this commented out? Are we now only keeping track of acquired resources in `local_available_resources_`?

Edit: Ah, I see now that you're modifying the map in `Schedule`. Can you remove this commented out line?

Reply:
yup.